sdk/core/core-lro/README.md

```diff
@@ -1,18 +1,24 @@
 # Azure Core LRO client library for JavaScript
 
-Azure's Core LRO is a JavaScript library that provides an API that aims to
-allow the azure-sdk-for-js public libraries to implement fully featured pollers
-to manage Long Running Operations (LROs) with the Azure services.
+`@azure/core-lro` is a JavaScript library that allows developers to build pollers that keep track of Long Running Operations (LROs) that require consecutive calls to endpoints provided by the Azure services to update a local representation of the status of the remote operation until its completion.
```

Comment:
Too long a sentence. Consider breaking it up for ease of readability. Below is an option, but am not sure if it ends up mis-representing what we want to say. LRO experts should review

```suggestion
`@azure/core-lro` is a JavaScript library that allows developers to build pollers that keep track of Long Running Operations (LROs) against Azure services. Until completion, such operations require consecutive calls to Azure services to update a local representation of the remote operation status.
```

Reply:
Thank you for reviewing this!